mods/tuxemon/db/technique/amnesia.json

```diff
@@ -3,21 +3,33 @@
   "animation": "sparks_gold_alt",
   "category": "physical",
   "effects": [
-    "damage"
+    "statchange"
   ],
   "healing_power": 0,
   "icon": "",
   "is_area": false,
   "is_fast": false,
   "potency": 1,
   "power": 2.25,
+  "statdodge": {
+    "minvalue": 0, "maxvalue": 3, "iterator":"-", "overridetofull":false
```

Comment:
Remove `iterator` and `overridetofull` when they have their default values, as you already made them optional. It makes the JSON easier to read.